docs/notebooks/4.1 Setting up and Using PV Sites.ipynb

```diff
@@ -0,0 +1,521 @@
+      "text/plain": [
+       "<Figure size 1500x400 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# NOTE: this will consume an API request!\n",
+    "res = live.advanced_pv_power(resource_id)\n",
+    "timeseries[\"updated\"] = res.to_pandas()\n",
+    "pd.concat(\n",
+    "    timeseries.values(),\n",
+    "    keys=timeseries.keys(),\n",
+    "    axis=\"columns\",\n",
+    "    names=[\"Version\", \"Parameter\"]\n",
+    ").plot(figsize=[15, 4])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5fd7140-0cd0-41e4-986a-8c16ffd3ff0c",
+   "metadata": {},
+   "source": [
+    "## Adjusting Advanced PV Power Settings\n",
+    "\n",
+    "When comparing observed production to Solcast live PV estimates, if you find discrepancies you may be able to choose more appropriate [model settings](https://docs.solcast.com.au/?#0c9d3ccf-e2a4-4583-86a3-f89c8d658fde) to get estimates which better fit the observations.  We will give a couple of examples.\n",
```

Comment:
Same again with no hashes for api docs.

Distinguish between adjusting configuration (useful for better fitting estimates), and query parameters which are used to account for operational changes that affect generation.
You don't adjust the amount of snow to get a better estimate, you adjust the amount because you went and cleared the panels and know they are clear even if snow fell on them.

Reply:
done